### PR TITLE
Expand Continous Integration

### PR DESCRIPTION
**Pull request by kepler0**

### CODE REVIEW

**Comment by kepler0** [comment]
I found a failing test!

**Comment by kepler0** [comment]
Nevermind. Just a time out issues, weird.

**Comment by noseglid** [comment]
As much as I love the appveyor thingie it doesn't work now. I haven't looked to closely at it, but I think we need to modify some tests to have them run on windows.

I would love to get the travis changes in asap though. Would you mind splitting this into separate PRs?

**Comment by kepler0** [comment]
No problem. Later today.

File: .travis.yml [hunk 1]
```diff
@@ -12,11 +10,29 @@ notifications:
...
+    - ATOM_CHANNEL=stable
+    - ATOM_CHANNEL=beta
+
+addons:
+  apt:
+    packages:
+    - build-essential
+    - git
+    - libgnome-keyring-dev
+    - fakeroot
```
Comment (noseglid) [inline .travis.yml]: Why do we need to install these?

**Comment by noseglid** [comment]
I'll do this in branch `travis-matrix`. We can do appveyor somewhere else since that's quite a bigger task.

**Pull request closed without merging**